packages/notifi-react-example/src/NotifiCard/NotifiCard.tsx

```diff
@@ -55,6 +56,12 @@ export const NotifiCard: React.FC = () => {
           inputSeparators={inputSeparators}
           cardId="71562316475a4171ae9c980adaefab7d"
         ></NotifiSubscriptionCard>
+        <NotifiIntercomCard
+          inputLabels={inputLabels}
+          inputSeparators={inputSeparators}
+          cardId="1045f61752b148eabab0403c08cd60b2"
+        >
+        </NotifiIntercomCard>
```

Comment:
Can we please update to `<NotifiIntercomCard />` (no extra element)? And update `<NotifiSubscriptionCard />`?

Reply:
updated